README.md

```diff
@@ -25,35 +25,38 @@ For more details on how to use this package, check out example documentation bel
 
 | Project | Description |
 |---|---|
-| [apitte-fullstack](https://github.com/contributte/playground/tree/master/apitte-fullstack) | Fullstack application with example of all plugins. |
-| [apitte-micro](https://github.com/contributte/playground/tree/master/apitte-micro) | Micro application with direct dispatching. |
-| [apitte-simple](https://github.com/contributte/playground/tree/master/apitte-simple) | Simple middleware application. |
-| [apitte-nette-simple](https://github.com/contributte/playground/tree/master/apitte-nette-simple) | Simple Nette |
-| [contributte-api-router](https://github.com/contributte/playground/tree/master/contributte-api-router) | Starter project for ApiRouter. |
-| [contributte-console](https://github.com/contributte/playground/tree/master/contributte-console) | Example of writing own commands and how to install console-extra with all commands. |
-| [contributte-datagrid](https://github.com/contributte/playground/tree/master/contributte-datagrid) | Dockerized example of our datagrid. [DEMO](https://examples.contributte.org/packages/datagrid/) |
-| [contributte-event-dispatcher](https://github.com/contributte/playground/tree/master/contributte-event-dispatcher) | Example of dispatching and subscribing events. |
-| [contributte-forms-bootstrap](https://github.com/contributte/playground/tree/master/contributte-forms-bootstrap) | Example of contributte/forms-bootstrap usage. |
-| [contributte-live-form-validation](https://github.com/contributte/playground/tree/master/contributte-live-form-validation) | Example of live form validation. |
+| apitte-fullstack - moved to https://github.com/contributte/apitte-skeleton | Fullstack application with example of all plugins. |
```

Comment:
Tohle prosim uplne odstranit.

Reply:
Nechápu. mám odsrtranit všechny poznámky o přesunech?

Reply:
Ano, presne tak.

Reply:
Není to škoda? Když přijde programátor, který rád používal playground na prozkoumání, jak něco má fungovat, po delší době, tak už bude ztracený, kde má hledat a bude zbytečně psát dotazy.

Reply:
Neboj. Mam s tim dalsi plany.

Reply:
Bojím, znám i sebe Milane, takže se dokýžu vcítit do začátečníků